docs/troubleshooting/failed-schema-sync.rst

```diff
@@ -0,0 +1,11 @@
+Failed Schema Sync
+===================
+
+The total binary footprint of all ScyllaDB columns, tables, etc., must fit a single commit log segment size 
```

Comment:
Please provide links to the documentation where we describe the default value of the commit log segment size and show how it can be changed.
Overall I think there should be a limitations chapter, not just the trouble shooting. In the limitations, we should state various limits, this being the first, perhaps, documented one.

Reply:
At the moment, we don't have a documentation section that describes `commitlog_segment_size_in_mb`, so I can't add a link.
I've discussed it with @tzach and we're not going to add a limitations section for now.